### PR TITLE
feat: update crypto payments

### LINKED ISSUE #201: Prevent the user from double clicking the Pay with Metamask button
## Description

If the user double clicks the metamask button
then rejects the first payment and approves the second the frontend gets stuck.

## Expected behavior

TO properly proceed to the payment success screen

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #201

### CODE REVIEW

**Comment by kelsos** [comment]
There is nothing to be done at the moment as part of this PR but I am keeping it open until we are ready to deploy

**Comment by lukicenturi** [comment]
Added fix for #201

File: components/checkout/pay/CryptoPaymentForm.vue [hunk 1]
```diff
@@ -180,19 +231,19 @@ const css = useCssModule();
   @apply border mx-auto mt-2 mb-10;
 }
 
-.inputs {
-  @apply flex flex-col justify-center gap-6;
+.canvas {
+  @apply w-[228px] h-[228px];
```
Comment (tewshi) [inline components/checkout/pay/CryptoPaymentForm.vue:235]: can we replace we rems:

w-[14.25rem] h-[14.25rem]

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**